tools/manifest/sourcefile.py

```diff
@@ -864,11 +864,11 @@ def manifest_items(self):
                 )]
 
         elif self.name_is_multi_global:
-            globals = b""
+            globals = u""
```

Comment:
Nit: it's a bit inconsistent that some text strings are prefixed with `u` while the others are not. Shall we apply the future lint for string literals to `tools/` as well? @jgraham